packages/core/admin/admin/src/content-manager/components/AttributeFilter/hooks/useAllowedAttributes.js

```diff
@@ -14,29 +21,40 @@ const useAllowedAttributes = (contentType, slug) => {
     },
   ]);
 
-  const readPermissionForAttr = get(readPermissionsForSlug, ['0', 'properties', 'fields'], []);
-  const attributesArray = Object.keys(get(contentType, ['attributes']), {});
-  const allowedAttributes = attributesArray
-    .filter((attr) => {
-      const current = get(contentType, ['attributes', attr], {});
-
-      if (!current.type) {
-        return false;
-      }
-
-      if (NOT_ALLOWED_FILTERS.includes(current.type)) {
-        return false;
-      }
-
-      if (!readPermissionForAttr.includes(attr) && attr !== 'id' && !TIMESTAMPS.includes(attr)) {
-        return false;
-      }
-
-      return true;
-    })
-    .sort();
-
-  return allowedAttributes;
+  const canReadAdminUsers =
+    findMatchingPermissions(allPermissions, [
+      {
+        action: 'admin::users.read',
+        subject: null,
+      },
+    ]).length > 0;
+
+  const attributesWithReadPermissions = get(
+    readPermissionsForSlug,
+    ['0', 'properties', 'fields'],
+    []
+  );
```

Comment:
```suggestion
  const attributesWithReadPermissions = readPermissionsForSlug?.[0]?.properties?.fields ?? [];
```

Not sure if all parts of the path need to be optional but it would be great if we could continue to fade out lodash where applicable.